config/spryk/templates/Zed/Persistence/ZedPersistenceFactory.php.twig

```diff
@@ -8,7 +8,6 @@ use Spryker\Zed\Kernel\Persistence\AbstractPersistenceFactory;
 
 /**
  * @method \{{ organization }}\Zed\{{ module }}\{{ module }}Config getConfig()
- * @method \{{ organization }}\Zed\{{ module }}\Persistence\{{ module }}QueryContainerInterface getQueryContainer()
```

Comment:
please also check other twig templates, which can contain `@method` phpdoc

Reply:
I'm not sure `getRepository` and `getEntityManager` can be added on the fly here, since we need to be sure these classes were added, but Factory Spryk only has one `preSpyk` - `AddModule`. I'd better leave this stuff for `c:s:s` command, since it does the magic.